cnd/src/network/swarm.rs

```diff
@@ -0,0 +1,210 @@
+use crate::{
```

Comment:
I assume this has just been moved from `network.rs` without functional changes :) (d631262d3a5c97cebdb910afb083c817cd7a7352)

Reply:
Yes, was moved without functional changes.